iot-e2e-tests/common/pom.xml

```diff
@@ -97,6 +97,11 @@
             <version>1.18.8</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-storage-blob</artifactId>
```

Comment:
We got this dependency from the IoT Hub device client before, but now that the device client doesn't depend on it anymore, the e2e tests need to declare this dependency

Reply:
this is the track 2 version, right?

Reply:
Yes, but it may be a couple of minor versions behind the latest